src/XamlStyler.Extension.Windows.VS2019/XamlStyler.Extension.Windows.VS2019.csproj

```diff
@@ -95,7 +95,7 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.VisualStudio.SDK">
-      <Version>15.0.1</Version>
+      <Version>17.4.33103.184</Version>
```

Comment:
I have not been able to successfully load the extension on VS2019 with any version of the SDK package >15.0.1 (with a very unhelpful/generic error), which is why this particular package has been kept on a lower version for the VS2019 project. Perhaps I am missing something?

Reply:
Reverted back. Had the same issue. Sorry.